index.js

```diff
@@ -67,6 +67,45 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(event) {
+    toggleColor(event.target)
+})
+
```

Comment:
Nicely done! Using arrow functions (ES6 feature), you could also write this solution as shown below. This is the same exact functionality!
`header.addEventListener("click", () => toggleColor(header))`